apps/web/composables/useBlockManager/useBlockManager.ts

```diff
@@ -39,6 +39,7 @@ export function useBlockManager() {
       currentBlock.value = data.value.blocks[index];
       const { isEditing } = useEditor();
       isEditing.value = true;
+      window.scrollTo(0, 0);
```

Comment:
This doesn't seem like desirable behaviour to me. This change would mean that whenever I want to edit a block that appears later on the page, I have to scroll to check every change.

Let's have a discussion here please.